packages/storybook/src/schematics/configuration/lib-files/.storybook/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../tsconfig.json",
+  "exclude": ["../src/test.ts", "../**/*.spec.ts"],
+  "include": ["../src/**/*"]
+}
```

Comment:
As said in another comment, I had to add `types: ["node"]` but I'm not sure anymore why I needed it. Would it help to figure out the reason? 

`tsconfig.json` from the Storybook CLI: https://github.com/storybookjs/storybook/blob/next/lib/cli/generators/ANGULAR/template/.storybook/tsconfig.json